Cargo.toml

```diff
@@ -41,17 +44,29 @@ time = "0.3"
 tokio = { version = "1", default-features = false, features = ["fs"] }
 ureq = { version = "2.9", default-features = false, features = ["json", "tls"] }
 version = "3"
-zip-extract = { version = "0.1.3", default-features = false, features = ["deflate"] }
+zip-extract = { version = "0.1.3", default-features = false, features = [
+    "deflate",
+] }
+rfd = { version = "0.14.1", default-features = false, features = [
+    "tokio",
+    "xdg-portal",
+] }
 
 [dependencies.iced]
 version = "0.12.1"
 default-features = false
-features = ["web-colors", "advanced", "image", "tokio"]
+features = [
+    "web-colors",
+    "advanced",
+    "image",
+    "tokio",
+    "wgpu",       # tiny-skia backend panics with small text boxes, fixed in latest master.
```

Comment:
Do you have a link to the issue, and do you know if the fix has already been released? I'd like to avoid bouncing around between tiny-skia and wgpu with every release.

Reply:
> Do you have a link to the issue, and do you know if the fix has already been released? I'd like to avoid bouncing around between tiny-skia and wgpu with every release.

Looks like I was wrong, the issue hasn't been fixed yet. I'll change the comment.

https://github.com/iced-rs/iced/issues/2332